cmd/gateway-azure.go

```diff
@@ -599,15 +599,56 @@ func azureListBlobsGetParameters(p storage.ListBlobsParameters) url.Values {
 // As the common denominator for minio and azure is readonly and none, we support
 // these two policies at the bucket level.
 func (a AzureObjects) SetBucketPolicies(bucket string, policyInfo policy.BucketAccessPolicy) error {
```

Comment:
Though comments say that the "none" policy is supported, it seems like this function will return a `NotImplemented` error if the policy is not read-only.

Is it actually working?